data/json/items/tool_armor.json

```
@@ -2451,7 +2473,10 @@
       "type": "transform"
     },
     "flags": [ "VARSIZE", "FANCY", "ALLOWS_NATURAL_ATTACKS" ],
-    "armor": [ { "encumbrance": 8, "coverage": 95, "covers": [ "torso", "arm_l", "arm_r", "mouth" ] } ]
+    "armor": [
+      { "encumbrance": 8, "coverage": 95, "covers": [ "torso", "arm_l", "arm_r", "mouth" ] },
+      { "encumbrance": 8, "coverage": 100, "covers": [ "head" ], "specifically_covers": [ "head_throat" ] }
```

```suggestion
      { "encumbrance": 1, "coverage": 100, "covers": [ "head" ], "specifically_covers": [ "head_throat" ] }
```
8 encumbrance for covering the neck seems cruel